.circleci/config.yml

```diff
@@ -211,38 +204,38 @@ workflows:
       - lint-packages:
           requires:
             - compile-packages
-      - test-packages:
-          requires:
-            - compile-packages
-      - compile-examples:
+# EXITING BECAUSE OF MEM????
```

Comment:
Still need this?

Reply:
I redid this again so it should be fixed. Going to try and resolve the CircleCI issues tomorrow.